docs/how_tos/how_to_connect_to_a_remote_sonoff_zbbridge.md

```diff
@@ -0,0 +1,22 @@
+# How to connect to a remote Sonoff ZBBridge
+This how-to explains how to run Zigbee2MQTT with a comercial Sonoff ZBBridge Gateway.
+We will use a Sonoff ZBBridge Gateway with custom firmware to connect to a serial port over TCP.
+In this way you can use a simple premade Hub/Gateway flash it with custom firmware and then use it as your coordinator.
+
+WiFi-based Serial-to-IP bridges are not recommended for Silicon Labs EZSP adapters as the serial protocol does not have enough fault-tolerance to handle packet loss or latency delays that can normally occur over WiFi connections.
```

Comment:
Please make not recommended bold, also add the following:

Keep in mind that the EZSP support is currently **experimental**.